src/__tests__/ParseError-test.js

```diff
@@ -27,4 +27,27 @@ describe('ParseError', () => {
       code: 123,
     });
   });
+
+  it('message must be a string', () => {
+    /**
+     * error as object
```

Comment:
Could you please split this into 2 separate tests and remove the comments. If a comment is needed to explain a test that is usually an indication that it should be a separate test.

Reply:
sure thing